pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Related Issue
+- Link for the issue
+
+# Proposed Changes
+- Change 1
+- Change 2
+- Change 3
+
+# Additional Info
+- Additional content or any information
+
+# Checklist
+- [ ] Tested the code
+- [ ] Formatted the code using `npm run format`
+- [ ] Linted the code using `npm run lint`
+- [ ] Updated `README.md`
+
+# Screenshots
+
+## Original
+** Original Screenshot **
+
+## Updated
```

Comment:
```suggestion
### Updated
```

Reply:
I've updated the template to have level 2 headings instead of level 1 
Thanks for the suggestion @MadhavBahlMD